README.md

```diff
@@ -1,46 +1,64 @@
 
-## Available Scripts
+# Built With
 
-In the project directory, you can run:
+- Major Languages
+  - JavaScript
+  - React
+  - Redux
+  - HTML
+  - CSS
+  - Jest
 
-### `npm start`
+# Getting Started
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+To get a local copy up and running, follow these simple steps:
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Usage
 
-### `npm test`
+First copy this repo and clone it on your machine using this [link](https://github.com/Ibnballo1/corona-virus).
+Then cd into corona-virus folder.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Install
 
-### `npm run build`
+`npm install`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Start Server
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+`npm start`
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+# Authors
 
-### `npm run eject`
+:adult: Abdullateef Bello
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- GitHub: [@Ibnballo1](https://github.com/Ibnballo1/)
+- Linkedin: [Abdullateef Bello](https://www.linkedin.com/in/abdullateef-bello-1b8006228/)
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+# Contributing
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Ibnballo1/corona-virus/issues)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+# Live Demo
```

Comment:
- [ ] I see that your using Netlify to deploy your project however, kindly consider adding a link to your deployed app here in your README file so users can check your app